core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.c

```diff
@@ -702,9 +687,7 @@ static TEE_Result initialize_pmic(const void *fdt, int pmic_node)
 	stm32mp_put_pmic();
 
 	if (dt_pmic_is_secure())
```

Comment:
I don't fancy we rely of secure-status in `dt_pmic_is_secure()`

Reply:
We can address this in a dedicated P-R, it's out of the scope of this series that only intend to get rid of *plat-stm32mp1/shared_resources.c*.